### ISSUE TITLE
Let's Encrypt external bug (zope-component not found)

### ISSUE THREAD

**Issue by nachoparker** [nextcloud/nextcloudpi#230]
with the latest image, Let's Encrypt couldn't be built due to an external bug. The fix will be pushed as soon as it is available.

```
HTTPError: 404 Client Error: Not Found for url: https://www.piwheels.hostedpi.com/simple/zope-component/
```

**Comment by nachoparker** [comment]
This is the issue https://github.com/bennuttall/piwheels/issues/20

**Comment by nachoparker** [comment]
Used workaround from [here](https://github.com/certbot/certbot/issues/5138#issuecomment-333391771) while it gets fixed. 

Fixed in `v0.31.6`

**Comment by nachoparker** [comment]
Fixed upstream, no workarounds needed anymore https://github.com/bennuttall/piwheels/issues/20#issuecomment-340108171
